bin/ncp-update

```diff
@@ -20,7 +20,159 @@
   [[ -f /.ncp-image ]] || cd "$TMPDIR"         # update locally during build
 
   echo -e "Performing updates"
-  ./update.sh || exit 1
+
```

Comment:
let's leave `ncp-update` the way it was and move all this code to `update.sh`

Reply:
I wasn't sure about the correct position inside update.sh, so I created a distinct script of the backward mechanism so that it can be moved easier to the right place.